scripts/vagrant/zeppelin-dev/Vagrantfile

```diff
@@ -0,0 +1,42 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "hashicorp/precise64"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = false
```

Comment:
Understood. This shouldn't be an issue, as Vagrant will only warn about it - it won't update automatically, like this:
```
==> sparkrvm: Checking if box 'ubuntu/vivid64' is up to date...
==> hostname: A newer version of the box 'hashicorp/precise64' is available! You currently
==> hostname: have version '20150815.0.0'. The latest is version '20151120.0.0'. Run
==> hostname: `vagrant box update` to update.
```

Here the user will need to take an action to run the update, but at least they should be warned if there is any major security issue or similar.